lib/checkother.cpp

```diff
@@ -1614,7 +1614,8 @@ void CheckOther::checkIncompleteStatement()
         if (bailout)
             continue;
 
-        constStatementError(tok->next(), tok->next()->isNumber() ? "numeric" : "string");
+        const Token *nextTok = tok->next();
+        constStatementError(nextTok, nextTok->isNumber() ? "numeric" : "string");
```

Comment:
Here order of parameters evaluation doesn't matter because evaluating either of parameters doesn't affect evaluation of others. Technically it's an FP.